src/Conversion/ONNXToMhlo/CMakeLists.txt

```diff
@@ -2,11 +2,19 @@
 
 # Please keep in alphabetical order.
 
-install(TARGETS 
-  MhloDialect
-  MLIRMhloUtils
+install(TARGETS
+  ChloOps
   HloOpsCommon
-  )
+  MLIRMhloUtils
+  MhloDialect
+  MhloRegisterDialects
+  StablehloBase
+  StablehloOps
+  StablehloRegister
+  StablehloTypeInference
+
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}
```

Comment:
Are these intended to be installed where onnx-mlir is installed? If so, you should follow the onnx-mlir pattern for choosing installation paths.

Reply:
Yes, changed to `lib` instead of `${CMAKE_INSTALL_LIBDIR}`